src/core/AnimatedAlways.js

```diff
@@ -1,13 +1,22 @@
 import AnimatedNode from './AnimatedNode';
+import invariant from 'fbjs/lib/invariant';
 
 export default class AnimatedAlways extends AnimatedNode {
   _what;
 
   constructor(what) {
     super({ type: 'always', what: what.__nodeID }, [what]);
+    invariant(
+      what instanceof AnimatedNode,
+      `Reanimated: Animated.always node argument should be of type AnimatedNode but got ${what}`
```

Comment:
Be consistent. Add prefix `Reanimated` to both native and JS site.

Reply:
It is on both sides, I baked it in as a prefix for `errorMsg` in MapUtils, I can do basically the same for invariant, and refactor all of its usages throughout Reanimated, or simply remove prefixing it in Java.

Reply:
I don't know. It's fine I think